src/main/java/io/appium/java_client/MobileCommand.java

```diff
@@ -151,7 +156,10 @@
             postC("/session/:sessionId/appium/performanceData/types"));
         commandRepository.put(GET_PERFORMANCE_DATA,
             postC("/session/:sessionId/appium/getPerformanceData"));
-
+        commandRepository.put(START_RECORDING_SCREEN,
+                postC("/session/:sessionId/appium/startRecordingScreen"));
```

Comment:
There is a change in API [here](https://github.com/appium/appium-base-driver/blob/master/docs/mjsonwp/protocol-methods.md#appium-extension-endpoints), so update here accordingly as `start_recording_screen` or else it will throw unsupported command exception.